webui/src/pages/auth/login.jsx

```diff
@@ -23,18 +23,19 @@ const LoginForm = () => {
                 <Card className="login-widget">
                     <Card.Header>Login</Card.Header>
                     <Card.Body>
-                        <Form onSubmit={async (e) => {
+                        <Form onSubmit={(e) => {
                             e.preventDefault()
-                            try {
-                                await auth.login(e.target.username.value, e.target.password.value);
-                                setLoginError(null);
-                                const nextOrRoot = next && next !== router.route ? next : '/';
-                                router.push(nextOrRoot);
-                            } catch(err) {
-                                setLoginError(err);
-                            }
-                        }}>
+                            auth
+                                .login(e.target.username.value, e.target.password.value)
+                                .then(user => {
+                                    setLoginError(null);
+                                    return router.push((!!next) ? next : '/');
+                                })
+                                .catch(err => {
```

Comment:
What is "`err`"?  The golang is up outside of webui and down cmd or pkg :stuck_out_tongue_closed_eyes: